packages/firebase_in_app_messaging/firebase_in_app_messaging/example/pubspec.yaml

```diff
@@ -8,9 +8,9 @@ environment:
 dependencies:
   cupertino_icons: ^1.0.2
   firebase_analytics: ^10.1.1
```

Comment:
https://github.com/firebase/flutterfire/pull/10472/files#diff-21ed3552c47812b496298fbcdd3b59e7c913f057f4010af868b72104c299a205R7 This should be updated.